src/EpicClipboard.ahk

```diff
@@ -4,119 +4,284 @@
+Gui Add, Button, x105 y275 w60 h40 g1_RPG, 1 🔢
+Gui Add, Button, x165 y275 w60 h40 g2_RPG, 2 🔢
+Gui Add, Button, x225 y275 w60 h40 g3_RPG, 3 🔢
+
+; Pings Panel 
+Gui Add, GroupBox, x10 y320 w280 h105, Pings
+Gui Add, Button, x15 y340 w90 h40 gSummon_RPG, Summon 📦
+Gui Add, Button, x105 y340 w90 h40 gTimeFight_RPG, Time To Fight 🛡️
+Gui Add, Button, x195 y340 w90 h40 gJoin_RPG, Join 🍪
+Gui Add, Button, x15 y380 w90 h40 gFish_RPG, Fish 🐟 ;⚔
+Gui Add, Button, x105 y380 w90 h40 gChop_RPG, Chop 🎄 ;🎃
+Gui Add, Button, x195 y380 w90 h40 gCatch_RPG, Catch 🎺
+
+; Rewards Panel
+Gui Add, GroupBox, x10 y425 w280 h105, Rewards
+Gui Add, Button, x15 y445 w135 h40 gLootbox_RPG, 📦 Lootbox 📦
+Gui Add, Button, x150 y445 w135 h40 gVote_RPG, 🗳️ Vote 🗳️
+Gui Add, Button, x15 y485 w135 h40 gDaily_RPG, 📅 Daily 📅
+Gui Add, Button, x150 y485 w135 h40 gWeekly_RPG, 📆 Weekly 📆
+
+; Enchant Panel
+Gui Add, GroupBox, x10 y530 w280 h65, Enchant
+Gui Add, Button, x15 y550 w135 h40 gESword_RPG, ✨ Sword ✨
+Gui Add, Button, x150 y550 w135 h40 gEArmor_RPG, ✨ Armor ✨
+
+; Window Settings
 Gui, +Alwaysontop -MaximizeBox
-Gui Show, w300 h345, EPIC Clipboard
+Gui Show, w295 h600, EPIC RPG Clipboard
 Return
-
+
+; Command Lines 
 Profile_RPG:
     ControlFocus,, ahk_exe Discord.exe
     ControlSend,, rpg p, ahk_exe Discord.exe
+    SendInput {enter}
     return
     
 Items_RPG:
     ControlFocus,, ahk_exe Discord.exe
     ControlSend,, rpg i, ahk_exe Discord.exe
+    SendInput {enter}
     return
     
 Hunt_RPG:
     ControlFocus,, ahk_exe Discord.exe
     ControlSend,, rpg %hunt_str%, ahk_exe Discord.exe
+    SendInput {enter}
     return
     
 Hunt_t_RPG:
     ControlFocus,, ahk_exe Discord.exe
     ControlSend,, rpg %hunt_str% together, ahk_exe Discord.exe
+    SendInput {enter}
     return
     
 Adv_RPG:
     ControlFocus,, ahk_exe Discord.exe
     ControlSend,, rpg %adventure_str%, ahk_exe Discord.exe
+    SendInput {enter}
     return
     
 Train_RPG:
     ControlFocus,, ahk_exe Discord.exe
     ControlSend,, rpg %training_str%, ahk_exe Discord.exe
+    SendInput {enter}
     return
     
-Ability_RPG:
+Work_RPG:
     ControlFocus,, ahk_exe Discord.exe
-    ControlSend,, rpg %farm_str%, ahk_exe Discord.exe
+    ControlSend,, rpg %work_str%, ahk_exe Discord.exe
+    SendInput {enter}
     return
     
+Farm_RPG:
+    ControlFocus,, ahk_exe Discord.exe
+    ControlSend,, rpg farm, ahk_exe Discord.exe
+    SendInput {enter}
+    return
+
+FarmC_RPG:
+    ControlFocus,, ahk_exe Discord.exe
+    ControlSend,, rpg %farm_str%, ahk_exe Discord.exe
+    SendInput {enter}
+    return
+
+Quest_RPG:
+    ControlFocus,, ahk_exe Discord.exe
+    ControlSend,, rpg quest, ahk_exe Discord.exe
+    SendInput {enter}
+    return
+
+EQuest_RPG:
+    ControlFocus,, ahk_exe Discord.exe
+    ControlSend,, rpg epic quest, ahk_exe Discord.exe
+    SendInput {enter}
+    return
+
+Duel_RPG:
+    ControlFocus,, ahk_exe Discord.exe
+    ControlSend,, rpg duel, ahk_exe Discord.exe
```

Comment:
Add a `space` and `@` to make it so you only have to type the name of the adversary

>ControlSend,, rpg duel @, ahk_exe Discord.exe

Reply:
This was something I was planning on updating today, implementing it in the next update.

Reply:
After trying to make the change to `ControlSend,, rpg duel @, ahk_exe Discord.exe`, it doesn't send `rpg duel @` but sends `rpg duel 2` instead. Am still new to AHK and not too sure why it is doing this, would be curious to know if you have a solution to this? For now I have left it to be `rpg duel` instead.

Reply:
That's weird, I tested before suggesting and it worked fine for me. Right, let's keep it as it is for now.